f5/bigip/__init__.py

```diff
@@ -34,19 +34,27 @@ def __init__(self, hostname, username, password, **kwargs):
         timeout = kwargs.pop('timeout', 30)
         allowed_lazy_attrs = kwargs.pop('allowed_lazy_attributes',
                                         allowed_lazy_attributes)
+        icontrol_version = kwargs.pop('icontrol_version', '')
```

Comment:
It seems like it would be more consistent with e.g. 'device_name' if we made this 'None'.